docs/examples/upload/avatar.vue

```diff
@@ -15,17 +15,17 @@
 import { ref } from 'vue'
 import { ElMessage } from 'element-plus'
 import { Plus } from '@element-plus/icons-vue'
+
 import type {
   UploadFile,
-  ElUploadProgressEvent,
-  ElFile,
-} from 'element-plus/es/components/upload/src/upload.type'
+  UploadProgressEvent,
+} from 'element-plus/es/components/upload'
```

Comment:
Could we import them from `element-plus`? Instead subpath.

Reply:
It doesn't work.